ExamplePlugins/ExampleNode/Model/CalculateTotalLengthCallback.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ExamplePlugins.ExampleNode.Model
+{
+    /// <summary>
+    /// An example class that is called into by <see cref="CalculateTotalLengthDfirNode"/> 
```

Comment:
I would be a little clearer that the DFIR node generates code that will call into this method at runtime by the execution system.